specification/2.0/README.md

```diff
@@ -831,57 +832,108 @@ A skin is instanced within a node using a combination of the node's `mesh` and `
+ 
+Each `image` defines an `image.sources` set of image sources. Type and format of image source are defined by `mimeType` and `internalFormat` specifiers.
+
+An image source can refer to image data either by array of URIs (external or base64-encoded), or by array of Buffer Views.
+
+> **Implementation Note** glTF 2.0 requires these arrays to contain no more than one element. This restriction could be lifted in future to support more texture targets.  
+ 
+First image pixel corresponds to the lower left corner of the image.
+
+> **Implementation Note**: With WebGL API, the first pixel transferred from the `TexImageSource` (i.e., HTML Image object) to the WebGL implementation corresponds to the upper left corner of the source. To achieve correct rendering, WebGL runtimes must flip Y axis by enabling `UNPACK_FLIP_Y_WEBGL` flag. This behavior was changed from glTF 1.0.
 
-Images referred to by textures are stored in the `images` array of the asset. Each image contains a URI to an external file (or a reference to a bufferView) in one of the supported images formats. Image data may also be stored within the glTF file as base64-encoded data and referenced via data URI. For example:
+Any colorspace information (such as ICC profiles, intents, etc) from PNG or JPEG containers must be ignored.
+
+> **Implementation Note**: This increases portability of an asset, since not all image decoding libraries fully support custom color conversions. To achieve correct rendering, WebGL runtimes must disable such conversions by setting `UNPACK_COLORSPACE_CONVERSION_WEBGL` flag to `NONE`.
+
+In the following example asset provides two versions of the same image: in sRGB and in linear colorspaces; so clients with hardware sRGB filtering support (or with enough fragment shader processing power) benefit from reduced banding, whilst low-power clients still get correct texel values by using linear-space image data directly.
 
 ```json
 {
     "images": [
         {
-            "uri": "duckCM.png"
-        },
-        {
-            "bufferView": 14,
-            "mimeType": "image/jpeg" 
+            "sources": [
+                {
+                    "internalformat": 32856,
+                    "mimeType": "image/png",
+                    "uris": ["duck_diffuse_linear.png"]
+                },
+                {
+                    "internalformat": 35907,
+                    "mimeType": "image/png",
+                    "uris": ["duck_diffuse_srgb.png"]
+                }
+            ]
         }
     ]
 }
 ```
-> **Implementation Note**: With WebGL API, the first pixel transferred from the `TexImageSource` (i.e., HTML Image object) to the WebGL implementation corresponds to the upper left corner of the source. Non-WebGL runtimes may need to flip Y axis to achieve correct texture rendering.
 
-### Samplers
+#### Image Formats
+
+glTF 2.0 allows a limited set of possible `internalformat` values based on OpenGL ES 3.0 enums.
```

Comment:
Isn't it possible that a glTF asset will contain a texture format that a WebGL 1.0 client will not be able to render?